include/baresip.h

```diff
@@ -46,6 +46,7 @@ struct vidsz;
 /** Defines the answermodes */
 enum answermode {
 	ANSWERMODE_MANUAL = 0,
+	ANSWERMODE_INTERCOM,
```

Comment:
perhaps it is better if the new mode is added to the end of the enum ?

Reply:
> perhaps it is better if the new mode is added to the end of the enum ?

Sure, can I do force push for fixing this?

Reply:
you can make the local change, then commit, then push.

when we merge we will squash all commits into one single nice commit :)